2017/devops-meetup-berlin-gitlabci/index.md

```diff
@@ -111,7 +111,7 @@ docker pull service-image:latest
 --
 
 ```
-# Explicitly tell docker to leverage the cache from the remote built image
+# Explicitly tell docker to leverage the cache from the remotely built image
```

Comment:
@bassco Thanks for this I am not 100% sure this change makes sense. The image isn't remotely built but the image is hosted remotely... 

I think a better change might be 

`Explicitly tell docker to leverage the cache from the registry image` 

Or 

`Explicitly tell docker to leverage the cache from the remote image`

Which do you think is better?

Reply:
Slight variation.... makes more sense given your context 

`Explicitly tell docker to leverage the cache from the image registry`